src/services/style-service.ts

```diff
@@ -63,6 +65,20 @@ const resolveColor = (theme: ExtendedTheme, color: PaletteColor | undefined) =>
 const fontSizeToCellHeight = (fontSize: string, lineClamp: number) =>
   +(parseInt(fontSize, 10) * LINE_HEIGHT_COEFFICIENT * lineClamp + CELL_PADDING_HEIGHT).toFixed(2);
 
+const resolveFontWeight = (fontStyleOptions: FontStyleOptions[] | undefined) => {
+  if (fontStyleOptions === undefined) {
+    return BOLD_FONT_WEIGHT;
+  }
```

Comment:
does this means the default font weight is bold for all headers?

Reply:
Yes, by default `fontStyleOptions` is `undefined`